pydocteur/github_api.py

```diff
@@ -24,37 +23,31 @@ def get_graphql_api(query: str) -> requests.Response:
     return resp
 
 
+def get_pull_request_from_check_run(commit_sha):
+    prs_for_commit = gh.search_issues(f"type:pr+repo:{REPOSITORY_NAME}+sha:{commit_sha}")
+    if prs_for_commit.totalCount != 1:
+        logger.error("Should be exactly one PR for this sha: %s, found %s", commit_sha, prs_for_commit.totalCount)
+    return prs_for_commit[0]
+
+
 def get_pull_request(payload):
-    logger.debug("Getting repository")
     gh_repo = gh.get_repo(REPOSITORY_NAME)
-    logger.info("Trying to find PR number from payload")
+    logger.info("Trying to find PR from %s", payload.get("action", "A payload with: " + ", ".join(payload.keys())))
 
-    is_run = payload.get("check_run", False)
-    is_suite = payload.get("check_suite", False)
+    head_sha = payload.get("check_run", {}).get("head_sha")
```

Comment:
You're only getting it from check_runs, why not check_suite ?

Reply:
Is it usefull?

Reply:
Or we should do the other way around: we should have less check_suite than check_run, but it would be enough?

Reply:
I also dropped check_run from the hooks, I bet we don't need to receive them.